bandit/reporters/info.h

```diff
@@ -274,6 +281,17 @@ namespace bandit {
         return std::string(2 * indentation_, ' ');
       }
 
+      void print_timing() {
+        if (report_timing_) {
+          std::chrono::duration<double> dur_in_sec(testcase_duration_);
+          stm_
+            << colorizer_.neutral()
+            << " ("
+            << std::to_string(dur_in_sec.count())
```

Comment:
I'll have to check this later, too, because I am not at all fluent in `std::chrono`. I know that it's just the code taken from the xunit reporter but what I *would* expect is the number of seconds but with precision in milliseconds, i.e. "0.123 seconds", not "0 seconds", not "0.12333333 seconds".

Reply:
Fixed here: 4e9ef0f

Has you can see I set the format through [std::fixed](http://www.cplusplus.com/reference/ios/fixed/). Is it fine for you?